scripts/test_nginx.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
```

Comment:
Most of this file is just boilerplate I've used with other web clients and can be ignored for the sake of this PR. `test_nginx.sh` doesn't run as part of the circle build so I just used it while developing the nginx config to verify it worked as expected.